frontend/src/components/WindSettings.jsx

```diff
@@ -20,391 +19,209 @@ const useStyles = makeStyles((theme) => ({
-                            <MenuItem value={val.value} key={val.id}>
-                                <em>{val.value}</em>
-                            </MenuItem>
-                            ))}
-                        </StyledSelect>
+            {!isWindSelectionEnabled && (
+                <Grid item container xs={12}>
+                    <Grid xs={10}
+                        classes={{ root: classes.backdropFilter }}
+                        sx={{ border: '1px white solid', textAlign: 'center' }}>
+                        <IconButton onClick={handleAddWindBlock} color="warning">
+                            <AddIcon />
+                        </IconButton>
                     </Grid>
                 </Grid>
+            )}
 
-            {/* WIND ORIGIN DROP DOWN */}
-            {/* <Grid item xs={3}> */} 
-                {/*<FormControl variant="standard" sx={{ minWidth: 150 }}>*/}
-                    {/*<InputLabel id="WindOrigin">Wind Origin</InputLabel>*/}
-                        {/*<Select*/}
-                        {/* label="Wind Origin"*/}
-                            {/*value={selectedWindOrigin}*/}
-                        {/* onChange={handleWindOriginChange}>*/}
-                        {/*{WindOrigin.map(function (val) {*/}
-                            {/*return (*/}
-                            {/* <MenuItem value={val.value} key={val.id}>*/}
-                            {/*     <em>{val.value}</em>*/}
-                            {/* </MenuItem>)*/}
-                            {/* })}*/}
-                {/* </Select>*/}
-                {/*</FormControl>*/}
-            {/*</Grid>*/}
-        
-                <Grid item container alignItems="center" direction="row">
-                    <Grid item xs={4}>
-                        <InputLabel id="velocity-label" sx={{ marginRight: 2, flexShrink: 0, color: '#F5F5DC', width: '200px' }}>
-                            Wind Velocity (m/s)
-                        </InputLabel>
+            {isWindSelectionEnabled && (
+                <>
+                    <Grid item container spacing={2} xs={12} classes={{ root: classes.backdropFilter, zIndex: 2 }}>
+                        {renderSelectField("Wind Type", selectedWindType, handleWindTypeChange, WindType)}
+                        {renderSelectField("Wind Direction", envConf.Wind.Direction, handleDirection, WindDirection)}
+                        {renderTextField("Wind Velocity (m/s)", envConf.Wind.Force, handleWindChange, { min: 0 })}
+
+                        {(selectedWindType === "Turbulent Wind" || selectedWindType === "Wind Shear") && (
+                            renderTextField("Fluctuation %", fluctuationPercentage, handleFLuctuationChange, {
+                                min: 0,
+                                max: 100,
+                                step: 0.1
+                            })
+                        )}
                     </Grid>
```

Comment:
@syedomair0 I think you can remove the above code from new line 176 to 189 as you already found a way to display multiple wind blocks using `map` function at new line 192.